axes/signals.py

```diff
@@ -27,16 +28,26 @@
 user_locked_out = Signal(providing_args=['request', 'username', 'ip_address'])
 
 
+def get_username_field():
+    try:
+        username_field = get_user_model().USERNAME_FIELD
```

Comment:
This works since django 1.7 I think we can safely asume it will work, so we don't need the try and except block